src/main/java/com/vmtapp/enterprise/dto/Error.java

```diff
@@ -2,8 +2,11 @@
 
 import lombok.Data;
 
+import java.util.Date;
+
 public @Data
 class Error {
     private String title;
     private String details;
+    private Date time;
```

Comment:
I suggest adding a Date field to this. It makes sense for error logging purposes and chronological sorting.